### PR TITLE
V3 documentation

### PR DESCRIPTION
**Pull request by Ceredron**

### CODE REVIEW

File: vipps-checkout-api.md [hunk 1]
```diff
@@ -197,13 +201,13 @@ With Vipps Checkout Direct you can easily implement an express checkout experien
 
 With Vipps Checkout Elements, you can adjust the fields and values present in the Checkout. For example, you might have a purchasing flow where you do not require an address because you are not sending physical goods, or you do not need the customer to identify themself because they are already logged into your system.
 
-The data collected can be adjusted according to your needs with the fields `addressFields` and `contactFields` in the session initiation request.
+The data collected can be adjusted according to your needs with the configuration.elements in the session initiation request.
 
-These fields are by default set to `true` and, if not specified, will return the full address and contact details in the Checkout session.
+If not specified, it will return the full address and contact details in the Checkout session.
```
Comment (sebdak) [inline vipps-checkout-api.md:206]:
```suggestion
If not specified, the complete address and contact details will be gathered.
```

Comment (sebdak) [reply vipps-checkout-api.md:206]: or "will be collected"?

File: vipps-checkout-api.md [hunk 2]
```diff
@@ -197,13 +201,13 @@ With Vipps Checkout Direct you can easily implement an express checkout experien
...
 
 #### AddressFields false example
 
-If you do not need the address from a user you can disable it, resulting in the following personal details form.
+If you do not need the address from a user you can disable it using Elements set to PaymentAndContactInfo, resulting in the following personal details form.
```
Comment (sebdak) [inline vipps-checkout-api.md:210]:
```suggestion
If you do not need the address from a user you can disable it using Elements set to `PaymentAndContactInfo`, resulting in the following personal details form.
```

File: vipps-checkout-api.md [hunk 3]
```diff
@@ -213,7 +217,7 @@ And the following payment form
 
 #### Addressfields and ContactFields false example
 
-If you do not need the contact details for a customer you can disable it, resulting in the following session
+If you do not need the contact details for a customer you can disable it using Elements PaymentOnly, resulting in the following session
```
Comment (sebdak) [inline vipps-checkout-api.md:220]:
```suggestion
If you do not need the contact details for a customer you can disable it using Elements `PaymentOnly`, resulting in the following session
```

**Review by sebdak — APPROVED** [review]
LGTM! minor comments

File: vipps-checkout-api-quick-start.md [hunk 4]
```diff
@@ -13,15 +15,15 @@ Use the Checkout API to create a checkout session, retrieve session information,
 
 ## Table of Contents
 
-* [Postman](#postman)
-  * [Prerequisites](#prerequisites)
-  * [Step 1: Get the Vipps Postman collection and environment](#step-1-get-the-vipps-postman-collection-and-environment)
-  * [Step 2: Import the Vipps Postman files](#step-2-import-the-vipps-postman-files)
```
[the remark this answers is not in the document]
Comment (Ceredron) [reply vipps-checkout-api-quick-start.md:19]: Tested and works the same.

[the remark this answers is not in the document]
Comment (Ceredron) [reply vipps-checkout-api-quick-start.md:19]: I originally thought I should change it back, but found the reasoning behind this autolint rule here:
https://github.com/arcticicestudio/styleguide-markdown/issues/3
TL;DR, asterisk is also used in other contexts such that is sort of an "overloaded" symbol. I agree with that reasoning.

[the remark this answers is not in the document]
Comment (SvenErikPy) [reply vipps-checkout-api-quick-start.md:19]: Fine by me!
